### DISCUSSION TITLE
Searching for an imsprog package sponsor

### DISCUSSION THREAD

**Discussion by bigbigmdm** [linuxmint/discussions#253] [Ideas]
I am looking for a sponsor to publish my package in Linux Mint.

[https://launchpad.net/imsprog](https://launchpad.net/imsprog)
repository ppa:bigmdm/imsprog
[GitHub](https://github.com/bigbigmdm/IMSProg)

Bug #1057386 [wnpp] Subject: ITP: imsprog -- linux chip programmer for CH341a devices

**Comment by Fantu** [comment]
the better is add in debian, ubuntu sync from debian and next mint versions will include it
I saw you did an ITP on debian, upload the package if you think near ready on https://mentors.debian.net/, on mentors you can receive reviews, if you need help you can also ask on the #debian-mentors irc channel
